### ISSUE TITLE
explorer doesn't switch to different explorer's url's when failing to connect

### ISSUE THREAD

**Issue by DylanVerstraete** [threefoldfoundation/tfchain-wallet-electron#22]

**Comment by GlenDC** [comment]
TODO: test and try to reproduce

**Comment by GlenDC** [comment]
Can reproduce easily:

```
var account = new window.tfchain.Account('foo', 'bar', null, 'testnet', ['https://explorer6.testnet.threefoldtoken.com', 'https://explorer5.testnet.threefoldtoken.com', 'https://explorer.testnet.threefoldtoken.com'])
undefined
account.explorer.blockchain_info_get().then(console.log)
Promise {<pending>}
tfchain.polyfill.http.py:12 GET https://explorer6.testnet.threefoldtoken.com/explorer net::ERR_NAME_NOT_RESOLVED
http_get @ tfchain.polyfill.http.py:12
(anonymous) @ tfchain.explorer.py:55
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:410
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:90
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ VM1111:1
org.transcrypt.__runtime__.py:271 retrying on another server, previous GET call failed: TypeError: Failed to fetch
tfchain.polyfill.http.py:12 GET https://explorer6.testnet.threefoldtoken.com/explorer/constants net::ERR_NAME_NOT_RESOLVED
http_get @ tfchain.polyfill.http.py:12
(anonymous) @ tfchain.explorer.py:55
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:410
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:90
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ VM1111:1
org.transcrypt.__runtime__.py:271 retrying on another server, previous GET call failed: TypeError: Failed to fetch
tfchain.polyfill.http.py:12 GET https://explorer6.testnet.threefoldtoken.com/explorer net::ERR_NAME_NOT_RESOLVED
http_get @ tfchain.polyfill.http.py:12
cb @ tfchain.explorer.py:68
Promise.catch (async)
catch_promise @ tfchain.polyfill.asynchronous.py:36
(anonymous) @ tfchain.explorer.py:70
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:410
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:90
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ VM1111:1
org.transcrypt.__runtime__.py:271 retrying on another server, previous GET call failed: TypeError: Failed to fetch
tfchain.polyfill.http.py:12 GET https://explorer6.testnet.threefoldtoken.com/explorer/constants net::ERR_NAME_NOT_RESOLVED
http_get @ tfchain.polyfill.http.py:12
cb @ tfchain.explorer.py:68
Promise.catch (async)
catch_promise @ tfchain.polyfill.asynchronous.py:36
(anonymous) @ tfchain.explorer.py:70
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:410
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:90
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ VM1111:1
org.transcrypt.__runtime__.py:271 retrying on another server, previous GET call failed: TypeError: Failed to fetch
tfchain.polyfill.http.py:12 GET https://explorer6.testnet.threefoldtoken.com/explorer net::ERR_NAME_NOT_RESOLVED
http_get @ tfchain.polyfill.http.py:12
cb @ tfchain.explorer.py:68
Promise.catch (async)
catch_promise @ tfchain.polyfill.asynchronous.py:36
(anonymous) @ tfchain.explorer.py:70
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:410
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:90
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ VM1111:1
org.transcrypt.__runtime__.py:271 servers exhausted, previous GET call failed as well: TypeError: Failed to fetch
tfchain.polyfill.http.py:12 GET https://explorer6.testnet.threefoldtoken.com/explorer/constants net::ERR_NAME_NOT_RESOLVED
http_get @ tfchain.polyfill.http.py:12
cb @ tfchain.explorer.py:68
Promise.catch (async)
catch_promise @ tfchain.polyfill.asynchronous.py:36
(anonymous) @ tfchain.explorer.py:70
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:410
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ tfchain.client.py:90
(anonymous) @ org.transcrypt.__runtime__.py:4
(anonymous) @ VM1111:1
org.transcrypt.__runtime__.py:271 servers exhausted, previous GET call failed as well: TypeError: Failed to fetch
tfchain_test.html:1 Uncaught (in promise) Function {__class__: ƒ, __args__: Array(1), stack: "No stack trace available", _endpoint: "/explorer", _addresses: Array(3), …}
Promise.then (async)
(anonymous) @ VM1111:1

```

**Comment by GlenDC** [comment]
Will fix this Monday first thing.

**Comment by GlenDC** [comment]
Fixed by b7ca2a0.
